src/utils/net.ts

```diff
@@ -27,6 +27,12 @@ export async function sendRequest(
     method,
     headers,
     data,
+  }).catch(error => {
+    if (error?.response?.data) {
+      throw Error(JSON.stringify(error.response.data))
```

Comment:
I would provide a bit nicer Error 😅 At least format it like suggested. Maybe include path of the request as well?
```suggestion
      throw Error(`Request failed! Response: ${JSON.stringify(error.response.data)}`)
```